Source/Core/Primitives/Math/Point2.cs

```diff
@@ -102,104 +102,79 @@ public static float Distance(Point2 left, Point2 right)
 		/// </summary>
 		public static Point2 operator +(Point2 left, Point2 right)
 		{
-			left.X += right.X;
-			left.Y += right.Y;
-			return left;
+			return new Point2(left.X + right.X, left.Y + right.Y);
```

Comment:
Can you change the style of the constructor calls in this PR into a multi-line format? Especially with repeating patterns, I think this makes it a lot more readable and easier to spot typos. Consider this:

```csharp
return new Vector4(
	left.X + right.X,
	left.Y + right.Y,
	left.Z + right.Z,
	left.W + right.W);
```

Reply:
Done